### ISSUE TITLE
Consider removing XRWebGLLayer's `context` attribute

### ISSUE THREAD

**Issue by ddorwin** [immersive-web/webxr#679]
`XRWebGLLayer`'s `context` attribute only reflects the value that was passed into the object's constructor. As in https://github.com/immersive-web/webxr/issues/513, we should consider removing this value.

In this case, removing this unnecessary attribute may also simplify future extensions to other types of layers (at least for WebGPU). See https://github.com/immersive-web/webxr/issues/268#issuecomment-497865830.

**Comment by ddorwin** [comment]
Does the help wanted label indicate this is waiting for a PR? It's not clear whether a decision was made.

**Comment by toji** [comment]
Yes, sorry. To be clear: The editors support this change and are open to anyone posting a PR for it.

**Comment by bricetebbs** [comment]
Created #699 for this.

**Comment by bricetebbs** [comment]
Created a new PR #707 for this. Sorry.

**Comment by toji** [comment]
Closed by #707
